continuous-integration/dev-sec-ops/readme.md

```diff
@@ -0,0 +1,30 @@
+# DevSecOps
+
+## The concept of DevSecOps
+
+DevSecOps or DevOps security is about introducing security earlier in the life cycle of application development (a.k.a shift-left), thus minimizing the impact of vulnerabilities and bringing security closer to development team.
+Further information is available [here](https://github.com/microsoft/CSEPackagedSprints/tree/master/DevSecOps).
```

Comment:
Given this is an open source playbook that we share with our customers and those outside Microsoft I don't think we should include this link. It is not accessible by people who are not part of the Microsoft org, it 404s.